packages/react-scripts/config/webpack.config.js

```diff
@@ -213,9 +208,8 @@ module.exports = function (webpackEnv) {
       pathinfo: isEnvDevelopment,
       // There will be one main bundle, and one file per asynchronous chunk.
       // In development, it does not produce real files.
-      filename: isEnvProduction
-        ? 'static/js/[name].[contenthash:8].js'
-        : isEnvDevelopment && 'static/js/bundle.js',
+      filename: 'project44-voc.js',
```

Comment:
Can we use the name from the package.json here? Since this will be used in many different projects we don't want them all named voc.